src/Sockets/index.js

```diff
@@ -6,13 +6,12 @@ import type { Store } from 'redux';
 import { processNewScores, processNewMessages } from '../Actions/index';
 import store from '../store';
 
-// TODO: This endpoint needs to be an environemntal variable
-//const socket = openSocket('https://databraid.localtunnel.me');
-
-const socket = openSocket('http://localhost:4000');
+// TODO: Get the openSocket accept info from .env file
+const socket = openSocket('https://databraid.localtunnel.me');
+// const socket = openSocket(process.env.SOCKET_URL);
 
 // TODO: needs to be refactored to remove reference to the store.
```

Comment:
Yeah, this will surely break when this widget is moved into the dashboard. All referenced to the store need to be within component tree which is provided the store through the provider

Reply:
I could use some clarification around this, @HamidAghdaee

Reply:
The fix for this doesn't have to be part of this PR but think about what would happen when this file tries to load the store from where it's loading it from when this widget is placed within the dashboard. Is this the correct store?